### PR TITLE
Use ruff formatter instead of black

### PR DESCRIPTION
**Pull request by erlendvollset**
It mirrors black but runs much faster

## Description
Please describe the change you have made.

## Checklist:
- [ ] Tests added/updated.
- [ ] Documentation updated. Documentation is generated from docstrings - these must be updated according to your change.
  If a new method has been added it should be referenced in [cognite.rst](https://github.com/cognitedata/cognite-sdk-python/blob/master/docs/source/cognite.rst) in order to generate docs based on its docstring.
- [ ] Changelog updated in [CHANGELOG.md](https://github.com/cognitedata/cognite-sdk-python/blob/master/CHANGELOG.md).
- [ ] Version bumped. If triggering a new release is desired, bump the version number in [_version.py](https://github.com/cognitedata/cognite-sdk-python/blob/master/cognite/client/_version.py) and [pyproject.toml](https://github.com/cognitedata/cognite-sdk-python/blob/master/pyproject.toml) per [semantic versioning](https://semver.org/).

### CODE REVIEW

**Review by doctrino — APPROVED** [review]
Nice! been looking forward to this one :)

File: tests/tests_integration/test_api/test_datapoints.py [hunk 1]
```diff
@@ -421,10 +421,10 @@ def test_retrieve_outside_points__query_limit_plusminus1_tests(
         "start, end, exp_first_ts, exp_last_ts",
         # fmt: off
         [
-            (631670400000 + 1, 693964800000,     631670400000,           693964800000),
-            (631670400000,     693964800000,     631670400000 - WEEK_MS, 693964800000),
-            (631670400000,     693964800000 + 1, 631670400000 - WEEK_MS, 693964800000 + WEEK_MS),
-            (631670400000 + 1, 693964800000 + 1, 631670400000,           693964800000 + WEEK_MS),
+            (631670400000 + 1, 693964800000, 631670400000, 693964800000),
```
Comment (haakonvt) [inline tests/tests_integration/test_api/test_datapoints.py:424]: Let’s also remove the `fmt on/off `comments since it doesn’t seem to respect them.

Comment (erlendvollset) [reply tests/tests_integration/test_api/test_datapoints.py]: It does respect it actually. It was just an artefact of running black without proper config.

Comment (erlendvollset) [reply tests/tests_integration/test_api/test_datapoints.py]: Fixed!

Comment (erlendvollset) [reply tests/tests_integration/test_api/test_datapoints.py]: Hmm. It's supposed to https://docs.astral.sh/ruff/formatter/#format-suppression.

Comment (erlendvollset) [reply tests/tests_integration/test_api/test_datapoints.py:423]: Removed though.

**Pull request merged**